docs/appendices/release-notes/unreleased.rst

```diff
@@ -58,6 +58,8 @@ Changes
 - Introduced new optional ``RETURNING`` clause for :ref:`Update <ref-update>` to
   return specified values from each row updated.
 
+- Added the :ref:`encode(bytea, format) <scalar-encode>` and :ref:`decode(text, format) <scalar-decode>` string functions.
```

Comment:
minor nitpick: new changes should be put in top in the related sections. also we use a 80 char line limit in rst files.

Reply:
Fixed in the latest commit I squashed.